coinstacks/ethereum/api/src/controller.ts

```diff
@@ -72,8 +74,9 @@ export class Ethereum extends Controller implements BaseAPI, EthereumAPI {
       }, [])
 
       return {
-        pubkey: data.address,
         balance: data.balance,
+        nonce: Number(data.nonce ?? 0),
```

Comment:
is it always safe to assume undefined nonce means start at 0?

Reply:
Think it is more of just a typing things since `nonce` is optional as in only eth uses it for blockbook. Every address has a nonce, so my assumption was it should technically always return, but if it doesn't I am just assuming new address and should start at 0 in that case.